engine/apps/aliyun_dyvms/tasks.py

```diff
@@ -0,0 +1,81 @@
+import requests
+
+from apps.aliyun_dyvms.models import AliyunDyvmsCallStatuses
+from common.custom_celery_tasks import shared_dedicated_queue_retry_task
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json, LetterCase
+from django.conf import settings
+from celery.utils.log import get_task_logger
+from celery.exceptions import Retry
+from apps.base.utils import live_settings
+from aliyunsdkcore.client import AcsClient
+from aliyunsdkdyvmsapi.request.v20170525.QueryCallDetailByCallIdRequest import QueryCallDetailByCallIdRequest
+import datetime
+from typing import cast
+
+MAX_RETRIES = 1 if settings.DEBUG else 100
+
+logger = get_task_logger(__name__)
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class AcsResponse:
```

Comment:
This class is already defined in phone_provider.py. I propose to move it to separate file, for example call it aliyun_types.py to use it across your app.